composer.json

```diff
@@ -22,14 +22,15 @@
   "extra": {
     "wordpress-install-dir": "wordpress"
   },
+  "minimum-stability": "dev",
   "require": {
     "php": ">=5.6"
   },
   "require-dev": {
     "automattic/vipwpcs": "^2.3",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
     "php-coveralls/php-coveralls": "^2.5",
-    "phpcompatibility/phpcompatibility-wp": "^2.1",
+    "phpcompatibility/php-compatibility": "@dev",
```

Comment:
Turns out even the dev version of `phpcompatibility/phpcompatibility-wp` is locked at `"phpcompatibility/php-compatibility" : "^9.0",` https://github.com/PHPCompatibility/PHPCompatibilityWP/blob/dbb88b4f397fe2d52af984cae7714d7cd28d5bc3/composer.json#L26 while the PHP8 compatibility lints are only part of the untagged dev version and the latest stable 9.3.5 (released in December 2019) https://github.com/PHPCompatibility/PHPCompatibility/releases/tag/9.3.5 doesn't include them.

Reply:
Or use this hack https://github.com/PHPCompatibility/PHPCompatibilityWP/issues/40#issuecomment-1139623926